### ISSUE TITLE
Low contrast of Quick Setup button

### ISSUE THREAD

**Issue by Alexander-Wilms** [flathub/flathub.org#43]
The combination of forground color #4A90D8 and background color #C0CEDD results in quite a low contrast of 2.09:1.

This fails to conform to the Web Content Accessibility Guidelines, cf. https://webaim.org/resources/contrastchecker/?fcolor=4A90D8&bcolor=C0CEDD
